CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ permalink: /docs/en-US/changelog/
 ### Enhancements
 
 * Improved site provisioning messages
+* MariaDB upgraded too v10.5
```

Comment:
@tomjn My only critique... incorrect spelling of "to" for this context

Reply:
I'm on butterfly key mechanisms :(